src/client/src/containers/About/index.tsx

```diff
@@ -35,21 +39,31 @@ const About = ({ versions, intl }: Props) => {
   const { formatMessage } = intl;
   return (
     <div className={styles.container}>
-      <div className={styles.title}>{formatMessage(messages.about)}</div>
       <p className={styles.repo}>
         <a
           className={styles.link}
           target="_blank"
+          rel="noreferrer noopener"
           href={WEB_TEMPLATE_STUDIO_LINKS.REPO}
           onKeyUp={keyUpHandler}
         >
-          Web Template Studio
+          {formatMessage(messages.visitRepo)}
+        </a>
+        <a
+          className={styles.link}
+          target="_blank"
+          rel="noreferrer noopener"
+          href="https://github.com/Microsoft/WebTemplateStudio/issues"
```

Comment:
Should we use a constant for this link? We can do it in a different PR if u want

Reply:
yeah I have to do something else with the sidebar so I will address that in that pr instead since I need this one to go in first

Reply:
I think constant already exists, right? :)